snap/README.md

```diff
@@ -0,0 +1,45 @@
+# EdgeX USB Camera Device Service Snap
+[![edgex-device-usb-camera](https://snapcraft.io/edgex-device-usb-camera/badge.svg)](https://snapcraft.io/edgex-device-usb-camera)
+
+This directory contains the snap packaging of the EdgeX USB Camera device service.
+
+The snap is built automatically and published on the Snap Store as [edgex-device-usb-camera].
+
+For usage instructions, please refer to Device Camera section in [Getting Started using Snaps][docs].
+
+## Build from source
+Execute the following command from the top-level directory of this repo:
+```
+snapcraft -v
+```
+
```

Comment:
We should mention that building this snap remains "challenging" and at least offer some options. I don't think the given option "just works".

Reply:
As explained in another comment, we have been able to build it without problems locally and on CI servers. If there are any OS-specific challenges related to using snapcraft to build core22-based snaps, it should be raised on snapcraft issue trackers.